pkg/graph/schema.resolvers.go

```diff
@@ -17,24 +18,12 @@ import (
 )
 
 func (r *modelPlanResolver) CmsCenters(ctx context.Context, obj *models.ModelPlan) ([]models.CMSCenter, error) {
-	// TODO: We should probably have a better way to handle enum arrays
-	var cmsCenters []models.CMSCenter
-
-	for _, item := range obj.CMSCenters {
-		cmsCenters = append(cmsCenters, models.CMSCenter(item))
-	}
-
+	cmsCenters := models.ConvertEnums[models.CMSCenter](obj.CMSCenters)
```

Comment:
I thought about it a bit, but I think it would be better to handle in another PR. I'm not envisioning this method as the final solution for enum arrays, but a step in the right direction for now. I think this model is one of the few that is in an an array and exists in the `models` package vs the  auto generated `model` package. 
 
 Ideally, I'd like to tackle the final approach in [EASI-1982](https://jiraent.cms.gov/browse/EASI-1982) . I started to do some work towards that here, but I think it would make sense to do another take on that ticket, and if it isn't needed still, remove them and rely on auto-generated. I think the main reason is to not have any hand rolled types reference any auto-generated model types.